src/reader/mod.rs

```diff
@@ -821,20 +840,20 @@ trait XmlSource<'r, B> {
 
     /// Read input until processing instruction is finished.
     ///
-    /// This method expect that `<?` already was read.
+    /// This method expect that start sequence of a parser already was read.
     ///
-    /// Returns a slice of data read up to end of processing instruction (`>`),
-    /// which does not include into result (`?` at the end included).
+    /// Returns a slice of data read up to end of a chunk,  which does not include
+    /// into result.
```

Comment:
replace with

> Returns a slice of data read up to the end of a chunk, which is not included in the result

But it also now sounds a bit overgeneralized.  It's not clear what constitutes "the end of a chunk" - that depends on the type of parser obviously, but it could be clearer.

Maybe instead of "the end of a chunk" say "the end of the {object / article / a synonym that isn't element or entity since they already mean something in XML} being parsed"

I'm just giving suggestions, feel free to adjust if you see fit